datasets/jo/jo_sanctions/crawler.py

```diff
@@ -0,0 +1,121 @@
+from pathlib import Path
+from zavod import Context
+from zavod import helpers as h
+from lxml.html import HtmlElement
+from openpyxl import load_workbook
+from rigour.mime.types import XLSX
+
+AUDIT_FIELDS = [
+    "row_no",
+    "mother_name",
+    "birth_place",
+    "first_and_surname",
+    "country",
+    "doc_type",
+    "doc_no",
+    "doc_issue_auth",
+    "doc_issue_date",
```

Comment:
these can be added with `h.make_identification` which returns an https://www.opensanctions.org/reference/#schema.Identification and adds the number to the appropriate prop.

You can use a lookup to determine whether it's a passport with something like `is_passport = context.lookup_value("identification_type", doc_type, False, warn_if_unmatched=True)`